src/main/java/org/jenkinsci/test/acceptance/plugins/mstestrunner/MSTestRunnerBuildStep.java

```diff
@@ -0,0 +1,36 @@
+package org.jenkinsci.test.acceptance.plugins.mstestrunner;
+
+
+import org.apache.commons.lang.StringUtils;
+import org.jenkinsci.test.acceptance.po.AbstractStep;
+import org.jenkinsci.test.acceptance.po.BuildStep;
+import org.jenkinsci.test.acceptance.po.Control;
+import org.jenkinsci.test.acceptance.po.Describable;
+import org.jenkinsci.test.acceptance.po.Job;
+
+@Describable("Run unit tests with MSTest")
+public class MSTestRunnerBuildStep extends AbstractStep implements BuildStep  {
+
+    private final Control testFilesControl = control("testFiles");
+    private final Control categoriesControl = control("categories");
+    private final Control resultFileControl = control("resultFile");
+    private final Control cmdLineArgsControl = control("cmdLineArgs");
+    private final Control continueOnFailControl = control("continueOnFail");
+    
+    public MSTestRunnerBuildStep(Job parent, String path) {
+        super(parent, path);
+    }
+
+    public void configure(String testFiles, String resultFile, String categories, String cmdLineArgs, boolean ignoreFailingTests) {
```

Comment:
Please split this into separate methods or just expose the `Controls` so it can be configured independently (provided there is no reason to always set these in bulk).